flow_screen_components/lookupFSC/.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
need to take this environment-specific stuff out. not everyone uses vscode, for example.